code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -88,7 +87,23 @@
 	new /obj/item/weapon/extinguisher/mini(src)
 	update_icon()
 
+/obj/item/weapon/storage/belt/utility/chief
+	name = "Chief Engineer's toolbelt"
+	desc = "Holds tools, looks snazzy"
+	icon_state = "utilitybelt_ce"
+	item_state = "utility_ce"
 
+/obj/item/weapon/storage/belt/utility/chief/New()

```

Comment:
I think leaving this empty and a new `belt/utility/chief/full` for one with the tools would be better